3-enrich/scala-common-enrich/src/test/resources/iglu-schemas/com.snowplowanalytics.snowplow.enrichments/api_request_enrichment_config/jsonschema/1-0-0

```diff
@@ -0,0 +1,167 @@
+{
```

Comment:
If you need thse schemas temporarily for tests - put them into a separate commit marked with "temp".